ops/ansible/playbooks-ccs/rotate_bfd_postgres_user.yml

```diff
@@ -0,0 +1,26 @@
+#ansible-playbook rotate_bfd_postgres_user.yml --extra-vars "rds_database= master_user= master_pw= old_user= db_user= db_user_pw= rds_region="
+- hosts: localhost
+  vars:
+    rds_database: 
+    master_user: 
```

Comment:
Maybe consider storing these details (master user and pass) in the respective environment vault files and pull from there. Just a thought.

Reply:
I was under the impression that you could either pass them in via the command line or a vars file and Ansible would handle them either way. If that is not the case I definitely want to make sure it supports Ansible Vault. I'll follow up with you.

Reply:
You're correct, either way does work. I was thinking, I dont see this master user and pass stored in the vaults currently (would be good to add them for safe keeping) and then you could pull these values from there.